xbmc/pvr/PVRDatabase.cpp

```diff
@@ -788,10 +789,6 @@ bool CPVRDatabase::Persist(CPVRChannel &channel, bool bCommit)
 
   if (QueueInsertQuery(strQuery))
   {
-    /* update the channel ID for new channels */
-    if (channel.ChannelID() <= 0)
-      channel.SetChannelID(static_cast<int>(m_pDS->lastinsertid()));
-
```

Comment:
Why is this not needed anymore?

Reply:
I knew you would ask:

1. As we 're not insertung instantly, this does not work (never did) as `m_pDS->lastinsertid()` gets not increased.
2. SetChannelID is called later on, after all INSERTs are done. I double checked this both in debugger and db browser. => https://github.com/xbmc/xbmc/blob/master/xbmc/pvr/PVRDatabase.cpp#L659

Reply:
Cool